### ISSUE TITLE
Video quality problem. Video Look Worth than In Game

### ISSUE THREAD

**Issue by shinkuan** [keijiro/FFmpegOut#52]
Here is two picture, one is from game, the other is from FFmpegOut video:
![OUG](https://user-images.githubusercontent.com/35415788/84617734-77a02200-af02-11ea-81ae-36e7618e86d7.PNG)
![ING](https://user-images.githubusercontent.com/35415788/84617739-7969e580-af02-11ea-90e2-fa4f066e338e.PNG)

The in game one look much more better than the output video, why??

**Comment by keijiro** [comment]
Because it's compressed.

**Comment by shinkuan** [comment]
How can I not compress it?

**Comment by keijiro** [comment]
The main motive of using ffmpeg is to compress the video. If you don't need video compression, I'd recommend using Unity Recorder with the Image Sequence option.

**Comment by keijiro** [comment]
For the details of Unity Recorder, please see the following page of the manual:
https://docs.unity3d.com/Packages/com.unity.recorder@2.2/manual/RecorderImage.html

**Comment by shinkuan** [comment]
> Unity Recorder

But you can only use Unity Recorder in editor. Can’t use it in builds. ?

**Comment by keijiro** [comment]
Right. Then you can try one of these lossless codecs with ffmpeg: https://superuser.com/questions/347433/how-to-create-an-uncompressed-avi-from-a-series-of-1000s-of-png-images-using-ff

You can customize the ffmpeg arguments by modifying FFmpegPreser.cs: https://github.com/keijiro/FFmpegOut/blob/master/Assets/FFmpegOut/Runtime/FFmpegPreset.cs

**Comment by shinkuan** [comment]
I tried:
`case FFmpegPreset.UnCompressed:    return "-c:v r210";`
from


> https://superuser.com/questions/347433/how-to-create-an-uncompressed-avi-from-a-series-of-1000s-of-png-images-using-ff

But the output file has 0kb, it is broken.
Sorry this is my first time using FFmpeg, how should I do it?

**Comment by keijiro** [comment]
I'd recommend checking if the argument (`-c:v r210`) works correctly with ffmpeg from the command line.
